### ISSUE TITLE
[Question] What type of converter/adaptor/etc for POT should I choose for a local feature extraction network?

### ISSUE THREAD

**Issue by dedoogong** [openvinotoolkit/openvino#5780]
##### System information (version)
- OpenVINO => 2021.3
- Operating System / Platform => Ubuntu18
- Compiler => GCC
- Problem classification: POT
- Framework: TensorFlow/Pytorch (if applicable)
- Model name: ASLFeat/R2D2 (if applicable)

##### Detailed description
Hello, I've converted ASLFeat/R2D2 models to onnx and converted to XML/BIN with MO and run by implementing my own cpp inference code. Thanksfully, it shows x2 times faster speed~!

Now I want to quantize those models using POT~!

First, I studied POT/Accuracy Checker part of Openvino and saw there are a lot of converters/adaptors/annotation. 
But, in my case(Local Feature Extract network), I'm not sure which type of annotation, converter class I need to inherit. Maybe OpticalFlowAnnotation can be the first trial, but there's no related example. 

Plus, I'm confusing how can I implement my custom annotation converter because these kind of model has no explicit label(like segementation mask image or class id or any target box coordinate) as these models are trained by triplet loss or some homograph related loss. 

Is there any hint for the case of local feature network?

Thank you

##### Issue submission checklist

 - [ ] I report the issue, it's not a question
   
 - [x] I checked the problem with documentation, FAQ, open issues, Stack Overflow, etc and have not found solution
   
 - [ ] There is reproducer code and related data files: images, videos, models, etc.

**Comment by dkurt** [comment]
Hi! You may take a look at the simplified Python API. An example for classification network: https://github.com/likholat/openvino_quantization

**Comment by dedoogong** [comment]
Thank  you!
So I think I can't use the provided openvino's annotation classes as there is no anntation class for dense correspondence, but I still can use POT python API without Accuracy Checker step(or just using Default Quantization step). Right?
I hope OpticalFlow Annotation or a new Local Feature Correspondence Annotation class+usages are provided as a clear sample ASAP.
